pom.xml

```diff
@@ -161,8 +162,35 @@
             <argLine>-Xms128m -Xmx1024m</argLine>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>com.mycila</groupId>
```

Comment:
[This mycila plugin](http://code.mycila.com/license-maven-plugin/) is also used by Google Guice: https://github.com/google/guice/blob/master/pom.xml#L222

Initially I tried to use Maven's license plugin (https://www.mojohaus.org/license-maven-plugin/), but it required me to create template of license, which is too much for my use case.